### PR TITLE
COMMERCE-1612 Make sure that existing users are added to the new site

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test

File: commerce-discount-service/src/main/java/com/liferay/commerce/discount/internal/CommerceDiscountCalculationImpl.java [hunk 1]
```diff
@@ -144,8 +141,7 @@ public CommerceDiscountValue getProductCommerceDiscountValue(
 		}
 
 		SearchContext searchContext = buildSearchContext(
-			cpInstance.getCompanyId(),
-			commerceContext.getCommerceChannelGroupId(),
+			cpInstance.getCompanyId(), cpInstance.getGroupId(),
```
Comment (marco-leo) [inline commerce-discount-service/src/main/java/com/liferay/commerce/discount/internal/CommerceDiscountCalculationImpl.java]: @alecsloan the pInstance.getGroupId() is not the same of getCommerceChannelGroupId, why we are changing this?

Comment (alecsloan) [reply commerce-discount-service/src/main/java/com/liferay/commerce/discount/internal/CommerceDiscountCalculationImpl.java]: This commit shouldn't be in this pull. Removing and force-pushing

**Comment by alecsloan** [comment]
ci:test

**Pull request merged**